ouroboros-consensus/changelog.d/20231206_171848_nick.frisby_ChainSync_refactor.md

```diff
@@ -0,0 +1,26 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+<!--
+### Non-Breaking
+
+- A bullet item for the Non-Breaking category.
+
+-->
+
+### Breaking
+
+- Bundled up the arguments to the ChainSync client into new record types.
```

Comment:
I would mention the type names for quick grepping if needed.

Reply:
Done